deps/ngtcp2/ngtcp2/lib/ngtcp2_log.h

```diff
@@ -46,7 +46,7 @@ typedef struct ngtcp2_log {
      told. */
   ngtcp2_tstamp last_ts;
   /* user_data is user-defined opaque data which is passed to
-     log_pritnf. */
```

Comment:
This fix would need to be made in the upstream dependency. This change should be removed from this PR.